app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class LikesController < ApplicationController
+  def create
+    like = Like.where(post_id: params[:post_id], user_id: current_user.id).first
+    if like.nil?
+      like = Like.new(post_id: params[:post_id], user_id: current_user.id)
+      flash.now[:danger] = 'Some errors' unless like.save
+    end
+    redirect_to user_path(current_user)
+  end
+end
```

Comment:
[OPTIONAL]
- Create a destroy method for the Likes controller and implement a delete functionality for likes. This way a user can choose to unlike a post :smile: